php-onbuild/README.md

```diff
@@ -294,7 +294,10 @@ various config files" below).
 There are environment variables for adding config files.
 
 - NGINX_CONF_INCLUDE (defaults to nginx-app.conf)
-  This file will be inserted to the main nginx configuration.
+  This file will be inserted at the server block in the main nginx
+  configuration.
+- NGINX_CONF_HTTP_INCLUDE (defaults to nginx-http.conf)
+  This file will be inserted at the http block in the main nginx configuration.
```

Comment:
Maybe note that this is only the case for our default `nginx.conf` file. If they provide their own, it won't work unless they add the include line as well.